pulp_container/app/serializers.py

```diff
@@ -351,6 +351,13 @@ class ContainerDistributionSerializer(DistributionSerializer, GetOrCreateSeriali
         view_name_pattern=r"remotes(-.*/.*)?-detail",
         read_only=True,
     )
+    secure = serializers.SerializerMethodField(
+        help_text="Whether the distribution is served over HTTP (unsecure) or HTTPS (secure)."
```

Comment:
s/unsecure/insecure

Reply:
@ipanova They're synonyms and unsecure is more commonly used for non-living objects while insecure is more common in emotional context, but I'll change it to insecure if you prefer it.

Reply:
Not going to argue on this because I am not a native speaker but it really feels weird when reading it.
Results from google say it all:
`unsecure network` ---> About 852,000 results (0.22 seconds) 
`insecure network` --> About 32,900,000 results (0.25 seconds)